src/dfx/src/lib/extension/manager/execute.rs

```diff
@@ -0,0 +1,37 @@
+use super::ExtensionManager;
+use crate::lib::error::ExtensionError;
+use std::ffi::OsString;
+
+impl ExtensionManager {
+    pub fn run_extension(
+        &self,
+        extension_name: OsString,
+        params: Vec<OsString>,
+    ) -> Result<(), ExtensionError> {
+        let extension_name = extension_name.clone().into_string().map_err(|_e| {
```

Comment:
Is there really no useful information in the discarded `_e` errors in this method?

Reply:
https://doc.rust-lang.org/std/ffi/struct.OsString.html#method.into_string
> On failure, ownership of the original OsString is returned.

Reply:
OsString doesn't implement `Display`, therefore to display it in error msg we'd need to use `Debug` representation `{0:?}`

Reply:
I swapped it for now but let me know if you think it should be reverted